java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -655,11 +655,17 @@ public void dbsize() {
 
         // test dbsize with routing - flush the database first to ensure the set() call is directed to a
         // node with 0 keys.
-        assertEquals(OK, clusterClient.flushall().get());
+        assertEquals(OK, clusterClient.flushdb().get());
         assertEquals(0L, clusterClient.dbsize().get());
+
         String key = UUID.randomUUID().toString();
+        SingleNodeRoute route = new SlotKeyRoute(key, PRIMARY);
         assertEquals(OK, clusterClient.set(key, "foo").get());
-        assertEquals(1L, clusterClient.dbsize(new SlotKeyRoute(key, PRIMARY)).get());
+        assertEquals(1L, clusterClient.dbsize(route).get());
+
+        // flush DB with route and check
+        assertEquals(OK, clusterClient.flushdb(route).get());
+        assertEquals(0L, clusterClient.dbsize(route).get());
     }
```

Comment:
Should we also test the flushdb calls that take a FlushMode argument here or is it unnecessary? Just bringing it up since the review checklist mentions testing all new command function signatures